packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -347,6 +349,8 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
             </Stack.Item>
             <SidePane
               mobileView={props.mobileView}
+              /* @conditional-compile-remove(video-background-effects) */
+              maxWidth={isVideoPaneOpen ? '19.5rem' : undefined}
```

Comment:
Does the maxwidth number `19.5rem` is provided by design or we are maitaining max height from our side, just curious :)

Reply:
Yeah, should've been 17.5 though my bad! Updated